### ISSUE TITLE
size matter

### ISSUE THREAD

**Issue by soheilmajidi** [yingkaisha/keras-unet-collection#38]
hello 
A question 
I have trained swin net model for segmentation and my image size was 64*64*3 ;
Can I use (prediction) this model (model and it's weights) when I have 256*256*3 or 256*256 without resizing it to 64*64*3 or not?
If answer is yet, how?

I tried but couldn't find a way except size decreasing and this will harm the resolution ?
I will be appreciated if you help me
Thanks ,

**Comment by yingkaisha** [comment]
No. Swin-Transformer accepts fixed input sizes only. If you configure them as 64-by-64, then they cannot make predictions on 256-by-256 inputs.
